src/Rings/MPolyQuo.jl

```diff
@@ -600,13 +610,13 @@ julia> R, (x,) = PolynomialRing(QQ, ["x"]);
 julia> A, p = quo(R, ideal(R, [x^4]));
 
 julia> f = p(x-x^6)
--x^6 + x
+x
```

Comment:
The `simplify` command seems to now be carried out automatically. Whoever is responsible for that: Maybe they wish to also adjust the documentation?

Reply:
It seems that `simplify` was called by some internal passage through the projection map to the quotient. But now that I changed it back, it's also back to the old version.